extensions/amp-analytics/0.1/test/test-requests.js

```diff
@@ -48,7 +47,8 @@ describes.realWin('Requests', {amp: 1}, env => {
       it('should batch multiple send', function* () {
         const spy = sandbox.spy();
         const r = {'baseUrl': 'r2', 'batchInterval': 1};
-        const handler = new RequestHandler(ampdoc, r, preconnect, spy, false);
+        const handler = new RequestHandler(ampdoc, r, preconnect, spy, false,
+            ampdoc);
```

Comment:
last param is an element?

Reply:
Fixed to return a mock that contains getAmpDoc() which is all we really need.